docs/content/connector/connector_protocol.md

```diff
@@ -78,12 +76,11 @@ Parameters can be specified during connector publication.
 The connector has to respond with one of the following status codes: **201**, **202**, **401** or **403**.
 Optionally the response can contain a payload specifying further steps required for the installation.
 The content of field **furtherStep** will be shown in the [Developer Center](https://devcenter.connctd.io/), formatted according to its type.
-The field **details** can contain an arbitrary JSON object.
+The field **details** can contain an arbitrary JSON object that will be rendered during the the installation process.
 If the connector responds with status code **202** and further steps, the installation state will be set to **ONGOING**.
 In that case, it is the responsibility of the connector to update the installation state as soon as the installation is finished.
 To update the installation state the connector must send a request to the appropriate [API endpoint](#update-installation-state) described below.
-<!-- TODO What is "details" used for? -->
-<!-- TODO Add an example on how the payload / further step is used -->
+This can be handy if the connector needs further authorization via e.g. an OAuth flow.
```

Comment:
```suggestion
This is useful in cases where the installation needs to be completed asynchronously because it was waiting for e.g. the users consent during an oauth2 flow.
```